### PR TITLE
[Event] - Added Docker Community Meetup Event - Bengaluru

### PR DESCRIPTION
**Pull request by adithyaakrishna**
Signed-off-by: Adithya Krishna <adikrish@redhat.com>

**Description**

This PR adds a new talk and event regarding the upcoming in-person docker developer community meetup at Bengaluru

Meetup Link: https://www.meetup.com/Docker-Bangalore/events/285342797/

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 4c1e2d6c292b4d95e52284756f916e9a8eb24a9f at: https://62a22b5e0fe16c105302f1f5--layer5.netlify.app

**Comment by leecalcote** [comment]
@Nikhil-Ladha regarding the large file size of this image, do we still have an image bot enabled on this repo? I don't think so, since it isn't always desirable. 

@adithyaakrishna, can you use .webp or .jpg (given that is is largely a photo) to reduce from ~9MB to ~350k?

**Review by leecalcote — CHANGES_REQUESTED** [review]
Contents look good. Image is pending file size reduction.

**Comment by Nikhil-Ladha** [comment]
> @Nikhil-Ladha regarding the large file size of this image, do we still have an image bot enabled on this repo? I don't think so, since it isn't always desirable.

Seems like access is revoked. Shall I re-configure it to allow image optimizations?

**Comment by leecalcote** [comment]
@Nikhil-Ladha, no thank you. Often it trades off minimization for quality past the limit that I'm willing to tolerate (so much that visitors of the site will surely notice the poor quality).

**Comment by adithyaakrishna** [comment]
> @Nikhil-Ladha regarding the large file size of this image, do we still have an image bot enabled on this repo? I don't think so, since it isn't always desirable.
> 
> @adithyaakrishna, can you use .webp or .jpg (given that is is largely a photo) to reduce from ~9MB to ~350k?

Done Dona Done ✅

**Comment by l5io** [comment]
🚀 Preview for commit 5c94032f2938cdaff43aced546f132502d82945b at: https://62a239c67438661c84568cfb--layer5.netlify.app

**Comment by Chadha93** [comment]
@adithyaakrishna blog is not visible in the preview, is it due to change in image file type?

**Comment by Chadha93** [comment]
Oops, I missed it, 😅 Yes yes it's an event. Thanks @adithyaakrishna

**Comment by adithyaakrishna** [comment]
There's a small change 😶 

@leecalcote Should this be of a type Meetup or an Event?

**Comment by leecalcote** [comment]
> There's a small change 😶
> 
> @leecalcote Should this be of a type Meetup or an Event?

It's been a while, but let's go with a Meetup, please.

**Comment by l5io** [comment]
🚀 Preview for commit d13aeea1bb9d531ca46f98013b6db56eda46714d at: https://62a32e08ac4cbc2c58d49926--layer5.netlify.app

File: src/collections/events/docker-developer-community-meetup-2022/index.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+</p>
+
+Speakers:
+
+<ul>
+  <li>
+    <Link to="/community/members/adithya-krishna">Aditya Krishna Sharma</Link>
+  </li>
+  <li>
+    <Link to="https://linkedin.com/in/uravgkarthik">Karthik Ravishankar</Link>
```
Comment (leecalcote) [inline src/collections/events/docker-developer-community-meetup-2022/index.mdx:54]: General note: `<Link>` is for internal URLs only. `<a>` is used for external URLs.
Specific rule: Only links to community member profiles are allowed here. It'd be great to have a link to Karthik's social profile. It's something that he'll need to earn in the community first, though.

Comment (leecalcote) [inline src/collections/events/docker-developer-community-meetup-2022/index.mdx:54]:
```suggestion
  <li>
```

File: src/collections/events/docker-developer-community-meetup-2022/index.mdx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+title: "Composing Cloud Native Infrastructure with Docker Desktop and Meshery"
+date: 2022-06-11
+thumbnail: ./dockerBengaluru.webp
+eurl: https://www.meetup.com/Docker-Bangalore/events/285342797/
+type: Meetups
+event-category: talk
+published: true
+featured: true
+product: Meshery Extensions
```
Comment (leecalcote) [inline src/collections/events/docker-developer-community-meetup-2022/index.mdx:10]: @debo19 is this a resource category?

File: src/collections/events/docker-developer-community-meetup-2022/index.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+upcoming: true
+# Deck: N/A
+# Video: N/A
+---
+
+import { Link } from "gatsby";
+
+<p>
+  Join Layer5, In-person at <a href="https://www.meetup.com/Docker-Bangalore/events/285342797/">Docker Developer Community Meetup - Bengaluru</a> as
+  we share on the latest Meshery Docker Extension and how you can manage your cloud native applications using it.
```
Comment (leecalcote) [inline src/collections/events/docker-developer-community-meetup-2022/index.mdx:21]:
```suggestion
  we share on the Meshery Docker Extension and how you can manage your cloud native applications using it.
```

File: src/collections/events/docker-developer-community-meetup-2022/index.mdx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+</p>
+
+<br />
+
+<h2>
+  Talk:{" "}
+  <a href="https://www.meetup.com/Docker-Bangalore/events/285342797/">Composing Cloud Native Infrastructure with Docker Desktop and Meshery</a>
+</h2>
```
Comment (leecalcote) [inline src/collections/events/docker-developer-community-meetup-2022/index.mdx:29]:
```suggestion
<h2>Composing Cloud Native Infrastructure with Docker Desktop and Meshery</h2>
```

Removing duplicate link.

File: src/collections/events/docker-developer-community-meetup-2022/index.mdx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+---
+
+import { Link } from "gatsby";
+
+<p>
+  Join Layer5, In-person at <a href="https://www.meetup.com/Docker-Bangalore/events/285342797/">Docker Developer Community Meetup - Bengaluru</a> as
+  we share on the latest Meshery Docker Extension and how you can manage your cloud native applications using it.
+</p>
+
+<br />
```
Comment (leecalcote) [inline src/collections/events/docker-developer-community-meetup-2022/index.mdx:24]:
```suggestion
```

Removing unnecessary line break.

File: src/collections/events/docker-developer-community-meetup-2022/index.mdx [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+<h2>
+  Talk:{" "}
+  <a href="https://www.meetup.com/Docker-Bangalore/events/285342797/">Composing Cloud Native Infrastructure with Docker Desktop and Meshery</a>
+</h2>
+
+<br />
```
Comment (leecalcote) [inline src/collections/events/docker-developer-community-meetup-2022/index.mdx:31]:
```suggestion
```

File: src/collections/events/docker-developer-community-meetup-2022/index.mdx [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+<ul>
+<li>Get service mesh support for your Docker Compose apps: Import your Docker Compose apps. Configure and deploy them to Kubernetes and any service mesh</li>
+<li>Help you with single-click deployment of any service mesh: Supports 10 different service meshes to the fingertips of developers in connection with Docker Desktop’s ability to deliver Kubernetes locally.</li>
+<li>Detection of Kubernetes environments: Scan your kubeconfigs and select your current Kubernetes environment. Switch from one environment to another one</li>
+</ul>
+
+This talk can help the attendees to improve upon their K8s or docker deployments as well as help them in managing different service meshes simultaneously. I have a pretty basic knowledge of cloud-native stuff having worked with Kubernetes, Docker, Helm, and using some of the service meshes, trying them out for different stuff or applications. I have been using docker desktop for the past year and a half and I’m also a maintainer of Meshery and have contributed to several other open-source projects.
```
Comment (leecalcote) [inline src/collections/events/docker-developer-community-meetup-2022/index.mdx:43]:
```suggestion
This talk can help the attendees to improve upon their K8s or Docker deployments as well as help them in managing different service meshes simultaneously. I have a pretty basic knowledge of cloud native stuff having worked with Kubernetes, Docker, Helm, and using some of the service meshes, trying them out for different stuff or applications. I have been using Docker Desktop for the past year and a half. I’m also a maintainer of Meshery and have contributed to several other open source projects.
```

**Comment by leecalcote** [comment]
Great job on reducing the image file size. 👍

File: src/collections/events/docker-developer-community-meetup-2022/index.mdx [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+</p>
+
+Speakers:
+
+<ul>
+  <li>
+    <Link to="/community/members/adithya-krishna">Aditya Krishna Sharma</Link>
+  </li>
+  <li>
```
Comment (leecalcote) [inline src/collections/events/docker-developer-community-meetup-2022/index.mdx:49]:
```suggestion
  <li>Karthik Ravishankar
```

**Review by leecalcote — APPROVED** [review]
Good to go other than the question of `product` category. // @debo19

**Pull request merged**